susemanager-utils/testing/docker/scripts/schema_migration_reportdb_test_pgsql.sh

```diff
@@ -16,9 +16,9 @@ fi
 
 # Database schema creation
 
-rpm -ivh /root/uyuni-reportdb-schema-4.3.1-1.3.uyuni1.noarch.rpm \
-	 /root/susemanager-schema-4.3.8-1.3.uyuni1.noarch.rpm \
-	 /root/susemanager-schema-utility-4.3.8-1.3.uyuni1.noarch.rpm
+rpm -ivh /root/uyuni-reportdb-schema-*.rpm \
+	 /root/susemanager-schema-*.rpm \
+	 /root/susemanager-schema-utility-*.rpm
```

Comment:
No, we have multiple available AFAIK
Part of the test is, that we decide where we want to start with the migration.
This can be differ depening on which test we run.